File: oblig2/problem_c.m


N = 100;


T = linspace(.1,20, N);

C_v = zeros(N);

for i = 1:N-1
    C_v(i) = (3*exp(1/T(i))) / ((T(i)^2)*(1+3*exp(1/T(i)))^2);
end

plot(T, C_v)
xlabel('T');
ylabel('C_v');

pause()

File: oblig3/project_1/problem_c.py

import numpy as np
import matplotlib.pyplot as plt

# Setup lists
T_values = [1.0, 0.95, 0.9, 0.8]
V = np.linspace(0.4, 6.0, 100)

# Calculate pressure for a given T: 
for T in T_values:
    p = 8*T/(3*V - 1) - 3/(V**2)
    plt.plot(V, p)


# Plot:
plt.legend(['$\hat{T} = 1.0$', '$\hat{T}=0.95$','$\hat{T} = 0.9$', '$\hat{T} = 0.8$'])
plt.xlabel('Volume V')
plt.ylabel('Pressure P')
plt.show()

